### PR TITLE
Feature - SelectionMode Xml Attribute + Sample Upgrade

### PR DESCRIPTION
**Pull request by quentin41500**
- SelectionMode Xml Attribute
- Sample For All selection Mode

Fixes https://github.com/prolificinteractive/material-calendarview/issues/539

### CODE REVIEW

File: library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java [hunk 1]
```diff
@@ -309,6 +309,8 @@ public void transformPage(View page, float position) {
                     .setShowWeekDays(showWeekDays)
                     .commit();
 
+            setSelectionMode(a.getInteger(R.styleable.MaterialCalendarView_mcv_selectionMode, 0));
```
Comment (tevjef) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/MaterialCalendarView.java]:
```diff
- setSelectionMode(a.getInteger(R.styleable.MaterialCalendarView_mcv_selectionMode, 0));
+ setSelectionMode(a.getInteger(R.styleable.MaterialCalendarView_mcv_selectionMode, SELECTION_MODE_NONE));
```

File: sample/src/main/java/com/prolificinteractive/materialcalendarview/sample/SelectionModesActivity.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    implements OnDateSelectedListener, OnRangeSelectedListener {
+
+  private static final DateFormat FORMATTER = SimpleDateFormat.getDateInstance();
+
+  @BindView(R.id.calendar_view_single) MaterialCalendarView single;
+  @BindView(R.id.calendar_view_multi) MaterialCalendarView multi;
+  @BindView(R.id.calendar_view_range) MaterialCalendarView range;
+  @BindView(R.id.calendar_view_none) MaterialCalendarView none;
+
+
```
Comment (tevjef) [inline sample/src/main/java/com/prolificinteractive/materialcalendarview/sample/SelectionModesActivity.java]: remove extra space

File: sample/src/main/res/layout/activity_selection_modes.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    <CheckBox
+        android:id="@+id/calendar_mode"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="8dp"
+        android:checked="false"
+        android:text="Week Calendar Mode"
+        />
+
+
```
Comment (tevjef) [inline sample/src/main/res/layout/activity_selection_modes.xml]: remove extra space

**Comment by quentin41500** [comment]
@tevjef the pr has been updated, thank you for reviewing

**Pull request merged**